pkg/build/android.go

```diff
@@ -78,7 +95,7 @@ func (a android) build(params Params) (ImageDetails, error) {
 	}
 	defer imageFile.Close()
 
-	if err := a.embedImages(imageFile, buildDistDir, "boot.img", "dtbo.img", "vendor_kernel_boot.img",
+	if err := a.embedImages(imageFile, buildDistDir, "boot.img", "dtbo.img", buildCfg.VendorBootImage,
```

Comment:
Can `buildCfg.VendorBootImage` be empty? Same question applies to other uses of `buildCfg` parameters above.

Reply:
Good point; I've added validation rules for each of the parameters now.